### PR TITLE
just propagate the encoding and locale params instead of imposing global environment settings

### PR DESCRIPTION
**Pull request by fdemilde**

### CODE REVIEW

**Comment by smecsia** [comment]
@fdemilde 
Thanks for this, and sorry for reviewing this so late. Could you please squash your commits?

**Comment by fdemilde** [comment]
Hi,
I have squashed my commits into 1 (left the commit from donbeave as a seperate commit though)

Kind regards

File: src/main/java/ru/yandex/qatools/embed/postgresql/config/PostgresProcessOutputConfig.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
+package ru.yandex.qatools.embed.postgresql.config;
+
+import de.flapdoodle.embed.process.config.io.ProcessOutput;
+import ru.yandex.qatools.embed.postgresql.Command;
+
+/**
+ * @author <a href='mailto:alexey@zhokhov.com'>Alexey Zhokhov</a>
+ */
+public class PostgresProcessOutputConfig {
```
Comment (fdemilde) [inline src/main/java/ru/yandex/qatools/embed/postgresql/config/PostgresProcessOutputConfig.java]: added the suggested private constructor

**Comment by fdemilde** [comment]
Hi Ilya,
I hope this was enough squashing?

kind regards
Frederik

**Pull request merged**